internal/k8s/reconciler/manager.go

```diff
@@ -91,27 +90,26 @@ func NewReconcileManager(config ManagerConfig) *GatewayReconcileManager {
 	})
 
 	return &GatewayReconcileManager{
-		controllerName:        config.ControllerName,
-		logger:                config.Logger,
 		client:                config.Client,
 		consul:                config.Consul,
 		consulCA:              config.ConsulCA,
-		sdsHost:               config.SDSHost,
-		sdsPort:               config.SDSPort,
-		gatewayClasses:        NewK8sGatewayClasses(config.Logger.Named("gatewayclasses"), config.Client),
-		gatewayValidator:      validator.NewGatewayValidator(config.Client),
-		routeValidator:        validator.NewRouteValidator(resolver, config.Client),
-		namespaceMap:          make(map[types.NamespacedName]string),
 		consulNamespaceMapper: config.ConsulNamespaceMapper,
 		deployer:              deployer,
-		store:                 config.Store,
 		factory: NewFactory(FactoryConfig{
 			ControllerName: config.ControllerName,
 			Logger:         config.Logger,
 			Client:         config.Client,
 			Deployer:       deployer,
 			Resolver:       resolver,
 		}),
+		logger:           config.Logger,
+		gatewayClasses:   NewK8sGatewayClasses(config.Logger.Named("gatewayclasses"), config.Client),
+		gatewayValidator: validator.NewGatewayValidator(config.Client),
+		namespaceMap:     make(map[types.NamespacedName]string),
+		routeValidator:   validator.NewRouteValidator(resolver, config.Client),
+		sdsHost:          config.SDSHost,
+		sdsPort:          config.SDSPort,
+		store:            config.Store,
```

Comment:
These got difficult to track across code areas and PRs, so I sorted them alphabetically